src/connection/database/actions.js

```diff
@@ -70,9 +70,11 @@ export function signUp(id) {
       additionalSignUpFields(m).forEach(x => {
         if (x.get('type') !== 'textinfo') {
           if (x.get('isExtra')) {
-            params.user_metadata.extraFields[x.get('name')] = c.getFieldValue(m, x.get('name'));
```

Comment:
let's just put it into a single line
params.user_metadata.extraFields[x.get('name')] = c.getFieldValue(m, x.get('name')).trim() 
or
params.user_metadata.extraFields[x.get('name')] =
    c.getFieldValue(m, x.get('name')).trim()

Reply:
I try, I think it was automatic

Reply:
Yes, it is formatted as part of the build step. Interesting configuration to come back to later...